examples/us_street.js

```diff
@@ -8,7 +8,8 @@ const Lookup = SmartySDK.usStreet.Lookup;
 // const credentials = new SmartyCore.StaticCredentials(authId, authToken);
 
 // for client-side requests (browser/mobile), use this code:
-let key = process.env.SMARTY_EMBEDDED_KEY;
```

Comment:
I think that it's important we keep this line as it adds context to the comment above. Maybe there could be a comment added after, such as... `let key = process.env.SMARTY_EMBEDDED_KEY // "your key"`.
Also, is there any reason we couldn't freshen this up by changing `let -> const`? I don't think this variable needs to be mutable, does it?